forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt

```diff
@@ -4,7 +4,7 @@ Types:Legendary Creature Homunculus
 PT:1/1
 T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ DBDig | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, look at the top two cards of your library, then put one of them into your graveyard.
 SVar:DBDig:DB$ Dig | ValidTgts$ You | DigNum$ 2 | ChangeNum$ 1 | DestinationZone$ Graveyard | LibraryPosition2$ 0
-T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
+T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library,Hand | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
```

Comment:
but isn't that wrong? the trigger says only "Libary -> Graveyard", nothing on the card says Hand?

Reply:
Ups i was wrong, The Trigger Descriptions needs to be updated too